pkg/xray/provider_test.go

```diff
@@ -75,12 +75,19 @@ func testAccCreateRepos(t *testing.T, repo, repoType string, projectKey string)
 func testAccDeleteRepo(t *testing.T, repo string) {
 	restyClient := GetTestResty(t)
 
-	response, errRepo := restyClient.R().Delete("artifactory/api/repositories/" + repo)
-	if errRepo != nil || response.StatusCode() != http.StatusOK {
+	resp, err := restyClient.R().Delete("artifactory/api/repositories/" + repo)
+	if err != nil || resp.StatusCode() != http.StatusOK {
 		t.Logf("The repository %s wasn't removed", repo)
 	}
 }
 
+func dummyError(t *testing.T) *resty.Response {
+	restyClient := GetTestResty(t)
```

Comment:
You don't need to actually make a request to get the response.

Resty's `Response` uses the underlying `http.Response` object which is just a struct.

See https://pkg.go.dev/github.com/go-resty/resty/v2#Response, https://github.com/go-resty/resty/blob/master/response.go#L46, and https://pkg.go.dev/net/http#Response

So I think you can do something like this:

```go
rawResponse := http.Response{}{
    Status: http.StatusNotFound,
}
resp := resty.Response{}{
    RawResponse: &rawResponse,
}
```

Reply:
Implemented.